lmdeploy/vl/templates.py

```diff
@@ -81,22 +81,31 @@ async def async_collect_pil_images(
                 # 'image_url': means url or local path to image.
                 # 'image_data': means PIL.Image.Image object.
                 if item['type'] == 'image_url':
-                    url = item['image_url']['url']
-                    images.append(url)
+                    item_copy = item['image_url'].copy()
+                    try:
+                        url = item_copy.pop('url')
+                        images_with_kwargs.append([url, item_copy])
+                    except KeyError:
+                        logger.error(f'invalid format {message}')
```

Comment:
The log content is ambiguous. 
When it happens, we have no idea if the `url` is invalid or the `data` in the next try-except block is invalid

Reply:
Currently, we don't check input format for llm and vlm. If the client provides invalid inputs, it will get raised error `openai.InternalServerError: Internal Server Error` and the server won't stop.

For official openai service, it wll alwayse get an output regardless of whether the messages is valid or not.